src/collectors.m4

```diff
@@ -525,6 +526,15 @@ m4_define([_ARG_POSITIONAL_DOUBLEDASH], [m4_do(
 )])
 
 
+argbash_api([ARGBASH_INDICATE_SUPPLIED], [m4_do(
+	[_ARGBASH_INDICATE_SUPPLIED($@)],
+)])
+
+
+m4_define([_ARGBASH_INDICATE_SUPPLIED], [m4_do(
```

Comment:
You can define the body of the macro right away without the intermediate `_ARGBASH_INDICATE_SUPPLIED`. In this case, there is no benefit of the additional layer.

Reply:
I agree. Just following convention already used. Will merge.